packages/easy-coding-standard/src/Console/Output/ConsoleOutputFormatter.php

```diff
@@ -83,10 +103,11 @@ private function reportFileDiffs(array $fileDiffPerFile): void
         $i = 0;
         foreach ($fileDiffPerFile as $file => $fileDiffs) {
             $this->easyCodingStandardStyle->newLine(2);
-            $boldNumberedMessage = sprintf('<options=bold>%d) %s</>', ++$i, $file);
-            $this->easyCodingStandardStyle->writeln($boldNumberedMessage);
+            $i = $this->handleBoldNumberedMessageInFile($i, $file);
 
             foreach ($fileDiffs as $fileDiff) {
+                $i = $this->handleBoldNumberedMessageInFileDiff($i);
```

Comment:
it is handle new behaviour for bold numbered message can read from file moved to `php-code-%s.php`